game2_3.py

```diff
@@ -0,0 +1,88 @@
+"""Create a game where the computer picks a random word and the player has to guess that word.
+The computer tells the player how many letters are in the word.
+Then the player gets five chances to ask if a letter is in the word.
+ The computer can only respond with “yes” or “no”. Then, the player must guess the word.
+ """
+import random
+
+words = ["masa", "casa", "scaun", "tabla", "televizor", "mobila", "creion", "noptiera", "candelabru"]
+
+
+#random.choice()
+
+print("Welcome to the game!")
+word = random.choice(words)
+
+number_of_chars = len(word)
+print (number_of_chars)
+print("The word has: %d letters" %(number_of_chars))
+print (word)
+i = 0
+while i < 5:
+    letter = input("Say letter:")
+    for char in word:
+        if char == letter:
+            print("este")
+        else:
+            print("nu este")
+    i +=1
```

Comment:
This code is not working as requested. If the guessed letter is in the word you should print YES other wise NO, not to print a list of yes and now while checking each letter.

At the end if word was guessed user should be informed about it, otherwise after the 5 tries he should be able to enter a word.